extras/fixture/src/unity_fixture.c

```diff
@@ -93,6 +95,8 @@ void UnityTestRunner(unityfunction* setup,
         UnityMalloc_StartTest();
         UnityPointer_Init();
 
+        Unity.CurrentTestStartTime = clock();
```

Comment:
Not sure if this is the best way to capture the start time, but seemed sensible.